packages/smooth_app/lib/data_models/continuous_scan_model.dart

```diff
@@ -60,7 +60,7 @@ class ContinuousScanModel with ChangeNotifier {
       _states.clear();
       _latestScannedBarcode = null;
       await refreshProductList();
-      for (final String barcode in _productList.barcodes.reversed) {
```

Comment:
Does that fix #1139

Reply:
it fixs partially #1139,  the reordering issue is ok, but we still switch to the last card when returning from ranking page and product page, I will handled it in the scope of #1139